python/hidet/graph/ops/conv2d/conv2d.py

```diff
@@ -100,13 +121,28 @@ def __init__(self, data: TensorNode, weight: TensorNode, stride: List[int], dila
 
 
 class Conv2dOp(Operator):
-    def __init__(self, x: Tensor, w: Tensor, stride: Sequence[int], dilations: Union[int, Sequence[int]], groups: int):
+    def __init__(
+        self,
+        x: Tensor,
+        w: Tensor,
+        padding: Sequence[int],
+        pad_value: float,
+        stride: Sequence[int],
+        dilations: Union[int, Sequence[int]],
+        groups: int,
+    ):
         stride = normalize_stride(stride)
         dilations = normalize_dilations(dilations)
         super().__init__(
             inputs=[x, w],
-            attributes={'stride': stride, 'groups': groups, 'dilations': dilations},
-            task=Conv2dTask(input_like(x, 'x'), input_like(w, 'w'), stride, dilations, groups),
+            attributes={
+                'padding': padding,
+                'pad_value': pad_value,
```

Comment:
1. Is this attribute necessary? (I mean any operator in onnx/pytorch used it?). 
2. Better move this attribute to the end of the attribute list, as it is the least used one.

Reply:
torch supports padding mode, so this is not exactly the same. Removing for now.